artmanflow/web/html_utils.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+class HtmlUtils:
+    def __init__(self):
+        pass
+
+    _COLOR_REGEX = r'(\033\[\d*;*\d*m)'
+
+    _COLOR_STYLE_MAP = {
```

Comment:
I am sure there is a package to do this kind of stuff.

Reply:
Not sure, but probably (i did some brief search, and could not find it). This was lightweight enough to "reimplement it". Note the color map is basically the only thing, which can be external, the rest of this file is still custom, and the whole file is 100 lines long, with the license header and color map occupying most of the lines .